### PR TITLE
Release notes for NetSuite 8.3.2

### PR DESCRIPTION
**Pull request by martinpaoloni**

### CODE REVIEW

**Comment by martinpaoloni** [comment]
Please merge when v8.3.2 is released.

**Comment by martinpaoloni** [comment]
The connector was released. Please merge this PR.

File: modules/ROOT/pages/connector/netsuite-connector-release-notes-mule-4.adoc [hunk 1]
```diff
@@ -7,6 +7,30 @@ Support Category: Select
...
+[%header%autowidth.spread]
+|===
+|Software | Version
+|Mule Runtime | 4.0.0 and later
+|SuiteTalk API | 2017.2
+|===
+
+[NOTE]
+The NetSuite version is not the same as the SuiteTalk API version. Your account may run on a different NetSuite version  than the version of the the API you are using.
+
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/connector/netsuite-connector-release-notes-mule-4.adoc]: extra space between "version" and "than"

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/connector/netsuite-connector-release-notes-mule-4.adoc]: It's not clear to me whether "your account" refers to my anypoint platform account, my netsuite account or what. If you think it's clear to customers of the connector, disregard this comment.

Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-connector-release-notes-mule-4.adoc]: Will fix the first. It's not a bad idea to clarify.

**Pull request merged**